### PR TITLE
docs: Adding documentation for  cli command

### LINKED ISSUE #458: DOCS: Add docs for `workspace` in CLI
## Description

Add documentation for the `workspace` command in our CLI.

## Solution

- Create a file named `workspace.md` in [here](https://github.com/keyshade-xyz/keyshade/tree/develop/docs/cli)
- Check the [commands](https://github.com/keyshade-xyz/keyshade/blob/develop/apps/cli/src/commands/workspace.command.ts) and put up the docs accordingly
- You can take reference from [here](https://docs.keyshade.xyz/cli/profile)

### PR DESCRIPTION
**Pull request by YuvalBubnovsky**
## Description

This PR adds documentation for the `workspace` feature in the CLI, providing users with clear instructions on how to utilize this functionality effectively. <br />

Fixes #458 

## Dependencies

No new dependencies introduced.

## Future Improvements

Possibly adding examples for common use cases of the `workspace` feature in future documentation updates.

## Mentions

@rajdip-b

## Screenshots of relevant screens

N/A

## Developer's checklist

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-check on my work

**If changes are made in the code:**

- [ ] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [ ] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [ ] I have added relevant screenshots in my PR
- [ ] There are no UI/UX issues

### Documentation Update

- [x] This PR requires an update to the documentation at [docs.keyshade.xyz](https://docs.keyshade.xyz)
- [x] I have made the necessary updates to the documentation, or no documentation changes are required.

___

### **PR Type**
Documentation

___

### **Description**
- Added detailed documentation for the `workspace` command in the CLI, including descriptions and usage examples for subcommands such as `create`, `delete`, `export`, `get`, `list`, `search`, and `update`.
- Provided comprehensive details on command arguments and options to guide users in managing workspaces effectively.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>workspace.md</strong><dd><code>Add documentation for `workspace` CLI command and subcommands</code></dd></summary>
<hr>

docs/cli/workspace.md

<li>Added comprehensive documentation for the <code>workspace</code> CLI command.<br> <li> Detailed subcommands: <code>create</code>, <code>delete</code>, <code>export</code>, <code>get</code>, <code>list</code>, <code>search</code>, <br><code>update</code>.<br> <li> Included usage examples and options for each subcommand.<br> <li> Provided descriptions for command arguments and options.<br>

</details>

  </td>
  <td><a href="https://github.com/keyshade-xyz/keyshade/pull/464/files#diff-b05e29e0937dd69ff54edad8518a573758815266e1c67c038074d2fa4481dc2d">+180/-0</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Comment by YuvalBubnovsky** [comment]
@rajdip-b I feel like the bot's suggestion for adding the info about the `delete` command is wrong, from what I've seen in the code (apps/cli/src/commands/workspace/delete.workspace.ts), no such confirmation for deletion exists..

I also think the suggestion for adding usage examples to `list` is redundent, but if you think this is a good idea - I'll add those.

File: docs/cli/workspace.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+description: The `workspace` command
+---
+
+# `workspace`
+
+The `workspace` command is used to manage workspaces on keyshade. A workspace is a collection of projects, environments, secrets, variables, and roles. It has a set of subcommands:
+
+- [`create`](#create): This command is used to create a new workspace.
+- [`delete`](#delete): This command is used to delete an existing workspace.
```
Comment (rajdip-b) [inline docs/cli/workspace.md:10]: add - 'All data related to this workspace will be permanently lost.'

File: docs/cli/workspace.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+- [`export`](#export): This command is used to export all projects, environments, secrets, variables, and roles of a workspace.
```
Comment (rajdip-b) [inline docs/cli/workspace.md:11]: add - 'into JSON format'

**Review by rajdip-b — CHANGES_REQUESTED** [review]

**Comment by rajdip-b** [comment]
> @rajdip-b I feel like the bot's suggestion for adding the info about the `delete` command is wrong, from what I've seen in the code (apps/cli/src/commands/workspace/delete.workspace.ts), no such confirmation for deletion exists..
> 
> I also think the suggestion for adding usage examples to `list` is redundent, but if you think this is a good idea - I'll add those.

Hmm, I think you have a valid point in here. Cool, you can drop that suggestion!

**Comment by YuvalBubnovsky** [comment]
@rajdip-b 
added 'into JSON format' and rebased.

**Review by rajdip-b — APPROVED** [review]

**Pull request merged**